app/src/main/res/values/attrs.xml

```diff
@@ -27,6 +27,7 @@
     <attr name="toolbarDivider" format="reference"/>
     <attr name="menuCategoryText" format="reference"/>
     <attr name="preferenceSectionHeader" format="reference"/>
+    <attr name="menuItemButton" format="reference"/>
```

Comment:
We might also consider renaming this to `menuItemButtonColor` or `menuItemButtonTintColor`. I am not quite sure if the iconTintColorResource refer to the background or image color here, but this can definitely be made clearer in the naming and description.